Day 6/DisplayPost.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import Post from './Post'
+import Button from './components/Button';
+
+const DisplayPost = () => {
+

```

Comment:
The component name rarely / never starts with a verb. The name of the component is usually a Noun based on what it does. Ideally, the name should be.

Posts
PostList

Reply:
Resolved.